near-sdk/src/collections/lookup_map.rs

```diff
@@ -284,4 +284,22 @@ mod tests {
             assert_eq!(map.get(&key).unwrap(), value);
         }
     }
+
+    #[test]
+    fn test_debug() {
+        let mut rng = rand_xorshift::XorShiftRng::seed_from_u64(4);
+        let mut map = LookupMap::new(b"m");
+        let mut baseline = vec![];
+        for _ in 0..10 {
+            let key = rng.gen::<u64>();
+            let value = rng.gen::<u64>();
+            baseline.push((key, value));
+            map.insert(&key, &value);
+        }
+
+        assert_eq!(
+            format!("{:?}", map),
+            format!("LookupMap {{ key_prefix: {:?}, el: PhantomData }}", map.key_prefix)
+        );
```

Comment:
It doesn't make sense to me why you would add random values to test the debug impl? Esp in this case since the values aren't included in the debug output.